### PR TITLE
[669] Display properties keywords for Usage and Definition element node

### PR DESCRIPTION
**Pull request by jessymallet**
Bug: https://github.com/eclipse-syson/syson/issues/669

### CODE REVIEW

File: doc/content/modules/user-manual/pages/release-notes/2024.9.0.adoc [hunk 1]
```diff
@@ -138,6 +138,7 @@ The source of the edge (the UseCase or Requirement) can be reconnected to anothe
 - Add support for public import in direct edit specializations.
 It is now possible to specialize an element with a qualified name containing namespaces importing the required features.
 For example, `myAttribute : ISQ::MassValue` now correctly types the attribute with `ISQBase::MassValue`.
+- Display prefix keywords in label for Definition and Usage element node.
```
Comment (AxelRICHARD) [inline doc/content/modules/user-manual/pages/release-notes/2024.9.0.adoc:141]: element node -> graphical nodes

File: CHANGELOG.adoc [hunk 2]
```diff
@@ -101,6 +101,7 @@ The source of the edge (the UseCase or Requirement) can be reconnected to anothe
 - https://github.com/eclipse-syson/syson/issues/665[#665] [services] Support public import in direct edit specializations.
 It is now possible to specialize an element with a qualified name containing namespaces importing the required features.
 For example, `myAttribute : ISQ::MassValue` now correctly types the attribute with `ISQBase::MassValue`.
+- https://github.com/eclipse-syson/syson/issues/639[#669] [diagrams] Handle properties keywords in label of Definition and Usage element node.
```
Comment (AxelRICHARD) [inline CHANGELOG.adoc]: node -> nodes

File: CHANGELOG.adoc [hunk 3]
```diff
@@ -101,6 +101,7 @@ The source of the edge (the UseCase or Requirement) can be reconnected to anothe
 - https://github.com/eclipse-syson/syson/issues/665[#665] [services] Support public import in direct edit specializations.
 It is now possible to specialize an element with a qualified name containing namespaces importing the required features.
 For example, `myAttribute : ISQ::MassValue` now correctly types the attribute with `ISQBase::MassValue`.
+- https://github.com/eclipse-syson/syson/issues/669[#669] [diagrams] Handle properties keywords in label of Definition and Usage element node.
```
Comment (AxelRICHARD) [inline CHANGELOG.adoc:104]: element node -> graphical nodes

File: backend/services/syson-services/src/main/java/org/eclipse/syson/services/DiagramDirectEditListener.java [hunk 4]
```diff
@@ -216,6 +216,8 @@ public void exitAbstractPrefixExpression(AbstractPrefixExpressionContext ctx) {
                 usage.setIsAbstract(true);
             } else {
                 usage.setIsAbstract(false);
+                // A variation is always abstract,
```
Comment (AxelRICHARD) [inline backend/services/syson-services/src/main/java/org/eclipse/syson/services/DiagramDirectEditListener.java:219]: Please remove final `,`, here and everywhere else in this PR

File: CHANGELOG.adoc (hunk 3 above)
Comment (jessymallet) [reply CHANGELOG.adoc]: Done

File: backend/services/syson-services/src/main/java/org/eclipse/syson/services/DiagramDirectEditListener.java (hunk 4 above)
Comment (jessymallet) [reply backend/services/syson-services/src/main/java/org/eclipse/syson/services/DiagramDirectEditListener.java]: Done

File: doc/content/modules/user-manual/pages/release-notes/2024.9.0.adoc (hunk 1 above)
Comment (jessymallet) [reply doc/content/modules/user-manual/pages/release-notes/2024.9.0.adoc]: Done

File: CHANGELOG.adoc [hunk 5]
```diff
@@ -101,6 +101,7 @@ The source of the edge (the UseCase or Requirement) can be reconnected to anothe
 - https://github.com/eclipse-syson/syson/issues/665[#665] [services] Support public import in direct edit specializations.
 It is now possible to specialize an element with a qualified name containing namespaces importing the required features.
 For example, `myAttribute : ISQ::MassValue` now correctly types the attribute with `ISQBase::MassValue`.
+- https://github.com/eclipse-syson/syson/issues/669[#669] [diagrams] Handle properties keywords in label of Definition and Usage graphical node.
```
Comment (AxelRICHARD) [inline CHANGELOG.adoc:104]: you forgot the final `s` on node
I wanted `nodes`, not `node`

File: doc/content/modules/user-manual/pages/release-notes/2024.9.0.adoc [hunk 6]
```diff
@@ -138,6 +138,7 @@ The source of the edge (the UseCase or Requirement) can be reconnected to anothe
 - Add support for public import in direct edit specializations.
 It is now possible to specialize an element with a qualified name containing namespaces importing the required features.
 For example, `myAttribute : ISQ::MassValue` now correctly types the attribute with `ISQBase::MassValue`.
+- Display prefix keywords in label for Definition and Usage graphical node.
```
Comment (AxelRICHARD) [inline doc/content/modules/user-manual/pages/release-notes/2024.9.0.adoc:141]: you forgot the final s on node
I wanted nodes, not node

Comment (jessymallet) [reply doc/content/modules/user-manual/pages/release-notes/2024.9.0.adoc]: Done

**Pull request merged**